### ISSUE TITLE
Build a machine learning engine with better accuracy

### ISSUE THREAD

**Issue by sauln** [codeforgoodconf/black_holes#2]
From black_holes_backend created by [amjcosta](https://github.com/amjcosta) : codeforgoodconf/black_holes_backend#2  

It would be ideal to create an engine with greater than 95% accuracy. One way to accomplish this may be using tensorflow to build a neural net.
